src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/AssessorServiceApiClient.cs

```diff
@@ -1,210 +1,220 @@
-                    importLearnerDetailRequest);
-            }
-        }
-
-        public async Task UpdateLastRunStatus(UpdateLastRunStatusRequest updateLastRunStatusRequest)
-        {
-            using (var request = new HttpRequestMessage(HttpMethod.Post, $"/api/v1/schedule/updatelaststatus"))
-            {
-                await PostPutRequest(request, updateLastRunStatusRequest);
-            }
-        }
-
-        public async Task UpdateStandards()
-        {
-            using (var request = new HttpRequestMessage(HttpMethod.Post, $"/api/v1/standard-version/update-standards"))
-            {
-                // no retry as this should not run more than once a day, this is a background task
-                // which will return 202 immediately as it takes a long time to complete on production data
-                await PostRequestWithoutRetry(request);
-            }
-        }
-
-        public async Task RebuildExternalApiSandbox()
-        {
-            using (var request = new HttpRequestMessage(HttpMethod.Post, $"/api/v1/externalapidatasync/rebuild-sandbox"))
-            {
-                // no retry as this should not run more than once a day, this is a background task
-                // which will return 202 immediately as it takes a long time to complete on production data
-                await PostRequestWithoutRetry(request);
-            }
-        }
-
-        public async Task ImportLearners()
-        {
-            using (var request = new HttpRequestMessage(HttpMethod.Post, $"/api/approvals/update-approvals"))
-            {
-                // no retry as this should not run more than once a day, this is a background task
-                // which will return 202 immediately as it takes a long time to complete on production data
-                await PostRequestWithoutRetry(request);
-            }
-        }
-
-        public async Task RefreshProviders()
-        {
-            using (var request = new HttpRequestMessage(HttpMethod.Post, $"/api/providers/refresh-providers"))
-            {
-                // no retry as this should not run more than once a day, this is a background task
-                // which will return 202 immediately as it takes a long time to complete on production data
-                await PostRequestWithoutRetry(request);
-            }
-        }
-
-        public async Task UpdateStandardSummary()
-        {
-            using (var request = new HttpRequestMessage(HttpMethod.Post, "api/v1/oppfinder/update-standard-summary"))
-            {
-                // no retry as this should not run more than once a day, this is a background task
-                // which will return 202 immediately as it takes a long time to complete on production data
-                await PostRequestWithoutRetry(request);
-            }
-        }
-    }
-}
+﻿using Microsoft.Extensions.Logging;

```

Comment:
This is entire file has changes marked probably as you have disturbed the spaces - need to revert that so only your changes - with correct spaces are included

Reply:
There was TABS in my new code, but rest has spaces.
Could it be EOF line character? it is now CRLF.

Reply:
Yes original had CR for EOF line.